### PR TITLE
docs: validator updates

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the Palette CLI validator page with new flags.

## Changed Pages

💻 [Preview URL for Page](https://65ef2bcb0ff9fb70a18e98ca--docs-spectrocloud.netlify.app//palette-cli/commands/validator)

## Jira Tickets

🎫 [DOC-1100](https://spectrocloud.atlassian.net/browse/DOC-1100)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._
This is a release PR.

[DOC-1100]: https://spectrocloud.atlassian.net/browse/DOC-1100?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/palette-cli/commands/validator.md [hunk 1]
```diff
@@ -50,11 +54,13 @@ can install the Validator into an existing Kubernetes cluster by using the Helm
...
+| -------------- | --------------- | ----------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `-f`           | `--config-file` | Install the Validator using a configuration file (optional). Provide the file path to the configuration file.                                   | string   |
+| `-o`           | `--config-only` | Generate a configuration file without proceeding with an actual install. Default: false                                                         | boolean  |
+| `-h`           | `--help`        | Help with any command.                                                                                                                          | -        |
+| `-r`           | `--reconfigure` | Reconfigure Validator and plugins prior to the installation. The `--config-file` flag must be included. Default: false.                         | boolean  |
+| `-p`           | `--password`    | Update credentials provided in a configuration. This does proceed with installation. The `--config-file` flag must be included. Default: false. | boolean  |
```
Comment (yuliiiah) [inline docs/docs-content/palette-cli/commands/validator.md:63]:
```suggestion
| `-p`           | `--password`    | Update credentials provided in the configuration file. This does proceed with installation. The `--config-file` flag must be included. Default: false. | boolean  |
```

Comment (yuliiiah) [reply docs/docs-content/palette-cli/commands/validator.md:63]: > This does proceed with installation.

I'm not clear on what this applies to 🤔 Meaning, if you use this flag, the CLI proceeds with installation?

File: docs/docs-content/palette-cli/commands/validator.md [hunk 2]
```diff
@@ -79,6 +85,13 @@ Generate a configuration file without proceeding with an actual installation
 palette validator install --config-only
 ```
 
+Update credentials provided in a configuration file. This does proceed with installation but will prompt for new
+credentials.
```
Comment (yuliiiah) [inline docs/docs-content/palette-cli/commands/validator.md:89]:
```suggestion
Update credentials provided in the configuration file. This does proceed with installation but will prompt for new
credentials.
```

File: docs/docs-content/palette-cli/commands/validator.md [hunk 3]
```diff
@@ -140,6 +153,13 @@ validatorconfigs.validation.spectrocloud.labs    2023-11-09T21:02:12Z
 You can use the `kubectl` command to view the validation results. To review all the results collectively, use the
 `describe` command to display the `validationresults` CR.
 
+:::tip
+
+Use the `validator describe` command to view the validation results. The `validator describe` provides a more
```
Comment (yuliiiah) [inline docs/docs-content/palette-cli/commands/validator.md:158]:
```suggestion
Use the `validator describe` command to view the validation results. `validator describe` provides a more
```

File: docs/docs-content/palette-cli/commands/validator.md [hunk 4]
```diff
@@ -294,6 +314,13 @@ changed. Once you resolve the failure, the Validator will update the `Validation
 
 Use the `kubectl describe` command to view the validation results.
 
+:::tip
+
+Use the `validator describe` command to view the validation results. The `validator describe` provides a more
```
Comment (yuliiiah) [inline docs/docs-content/palette-cli/commands/validator.md:319]:
```suggestion
Use the `validator describe` command to view the validation results. `validator describe` provides a more
```

Comment (yuliiiah) [reply docs/docs-content/palette-cli/commands/validator.md:319]: Or _The `validator describe` command_ :)

File: docs/docs-content/palette-cli/commands/validator.md (hunk 3 above)
Comment (yuliiiah) [reply docs/docs-content/palette-cli/commands/validator.md:158]: Or _The `validator describe` command_ :)

**Review by yuliiiah — APPROVED** [review]
Approving with some copy-edits and small questions :) Good job ✨

File: docs/docs-content/palette-cli/commands/validator.md (hunk 3 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/palette-cli/commands/validator.md:158]: Ooo let's go with the later suggestion 😄

**Pull request merged**